src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -41,14 +50,14 @@ const NavHeader = (props) => {
     onCancelToken,
     placeholderText,
     setIsFirstTimeOpen,
-    getSearchLayout
+    getSearchLayout,
+    opacityImage
   } = props;
   const navigation = useNavigation();
   const coverPath = topicDetail?.cover_path || null;
-  const [domainHeight, setDomainHeight] = React.useState(0);
-  const [searchHeight, setSearchHeight] = React.useState(0);
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
+  const insets = useSafeAreaInsets();
+  const {width: displayWidth} = useWindowDimensions();
+  const isUserFollow = initialData?.isFollowing ? initialData.isFollowing : isFollow;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removed the callbacks `onDomainLayout` and `onSearchLayout` which were used to calculate heights for certain elements. If these calculations are no longer needed, this change is fine. However, if they are still required elsewhere, removing these could lead to issues. Please ensure that these changes do not affect other parts of the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->